### ISSUE TITLE
Set up staging dev server

### ISSUE THREAD

**Issue by ChurchOfTheSubgenius** [commune-os/commune-rs#49]
This server needs to be live on `rust.commune.sh` so we have a staging server of sorts to always have available for manual testing.

**Comment by avdb13** [comment]
Good idea. All endpoints are required for that I assume?

**Comment by ChurchOfTheSubgenius** [comment]
> Good idea. All endpoints are required for that I assume?

No, the existing server in it's current form should be deployed somewhere! Set up to reply on changes.
